cmd/csaf_checker/processor.go

```diff
@@ -340,6 +340,15 @@ func (p *processor) domainChecks(domain string) []func(*processor, string) error
 
 	if !direct {
 		checks = append(checks, (*processor).checkWellknownSecurityDNS)
+	} else {
+		p.badSecurity.use()
+		p.badSecurity.info(
+			"Performed no test of security.txt since the direct url of the provider-metadata.json was used.")
+		p.badWellknownMetadata.use()
+		p.badWellknownMetadata.info(
+			"Performed no test on whether the provider-metadata.json is available under the .well-known path since the direct url of the provider-metadata.json was used.")
+		p.badDNSPath.use()
+		p.badDNSPath.info("Performed no test on the contents of https://csaf.data.security.DOMAIN since direct url of provider-metadata.json was used")
```

Comment:
```suggestion
		p.badDNSPath.info("Performed no test on the contents of https://csaf.data.security.DOMAIN since direct url of provider-metadata.json was used.")